boards/px4/fmu-v2/init/rc.board

```diff
@@ -43,6 +43,7 @@ fi
 
 # External I2C bus
 hmc5883 -C -T -X start
+qmc5883 -C -T -X start
```

Comment:
This command fails to start the driver.  Still have to manually start at the command line.

Reply:
Fixed this issue in the probe() command for i2c.  Have to read 0x00 once first before you can read the ID registers reliably.

Reply:
I also added a while loop to check a number of retries.